### ISSUE TITLE
Unit tests for model manager

### ISSUE THREAD

**Issue by smrgeoinfo** [usgin/modelmanager#1]
Genhan mentioned that this had been identified as a gap

**Comment by rclark** [comment]
Yes, she did some work on testing but it is by no means complete.

**Comment by rclark** [comment]
@asonnenschein @jalisdairi Django is a really great environment to try writing your first tests.
